src/nvim/eval.lua

```diff
@@ -13377,18 +13377,20 @@ M.funcs = {
     args = 1,
     base = 1,
     desc = [=[
-      The result is a Number, which is the width of window {nr}.
-      {nr} can be the window number or the |window-ID|.
-      When {nr} is zero, the width of the current window is
-      returned.  When window {nr} doesn't exist, -1 is returned.
-      An existing window always has a width of zero or more.
+      Gets the width of |window-ID| {nr} (zero for "current
+      window"), or -1 if window {nr} doesn't exist. An existing
+      window always has a width of zero or more.
+
```

Comment:
This is inconsistent with `'winheight'`

Reply:
I assume you mean `winheight()`. I'll update that too.

Reply:
Oops